projects/ion/src/lib/input-area/input-area.component.scss

```diff
@@ -0,0 +1,86 @@
+@import '../../styles/index.scss';
+
+@mixin add-colors() {
+  border-color: $primary-5;
+  outline: 2px solid;
+  outline-color: $primary-2;
+}
+
+textarea {
+  color: $neutral-7;
+  padding: spacing(1.3) spacing(1.5);
+  border: 1px solid $neutral-5;
+  border-radius: 8px;
+  background-color: $neutral-1;
+  resize: none;
+  outline: none;
+  cursor: text;
+  overflow-y: auto;
+
+  &:hover {
+    border-color: $primary-4;
+  }
+
+  &:focus {
+    @include add-colors();
+  }
+
+  &:active {
+    @include add-colors();
+  }
+}
+
+::placeholder {
+  color: $neutral-5;
+  font-weight: 400;
+  font-size: 14px;
+}
```

Comment:
I'm a little afraid that this particular code may affect other components. When you have more components with a placeholder and they are all in DOM, it may affect everything, not just the text area component.
Maybe it's better to be safe and put this little code inside `textarea {}` scope above.